utility/importer/brave_profile_import_impl.cc

```diff
@@ -23,11 +59,54 @@ void BraveProfileImportImpl::StartImport(
   } else if (base::StartsWith(base::UTF16ToUTF8(source_profile.importer_name),
                               "Chromium", base::CompareCase::SENSITIVE)) {
     command_line->AppendSwitch("import-chromium");
-  } else if (base::StartsWith(base::UTF16ToUTF8(source_profile.importer_name),
-                              "Brave", base::CompareCase::SENSITIVE)) {
-    command_line->AppendSwitch("import-brave");
   }
 
-  ProfileImportImpl::StartImport(
-      source_profile, items, localized_strings, std::move(observer));
+  content::UtilityThread::Get()->EnsureBlinkInitialized();
```

Comment:
what is this needed for?

Reply:
Well, I just copied from original code from `ProfileImportImpl::StartImport`.
W/o this, we got DCHECK failure during the import. Maybe due to using blink API w/o blink initialization.
```
[90878:17667:0620/085236.166224:FATAL:partitions.h(98)] Check failed: initialized_.
0   libbase.dylib                       0x0000000104475e79 base::debug::CollectStackTrace(void**, unsigned long) + 9
1   libbase.dylib                       0x000000010432a343 base::debug::StackTrace::StackTrace() + 19
2   libbase.dylib                       0x000000010434e139 logging::LogMessage::~LogMessage() + 249
3   libwtf.dylib                        0x0000000116f57804 WTF::Partitions::FastMalloc(unsigned long, char const*) + 100
4   libblink_platform.dylib             0x000000011d6cdb69 blink::WebData::Assign(char const*, unsigned long) + 41
5   libcontent.dylib                    0x000000010df550ed content::DecodeImage(unsigned char const*, gfx::Size const&, unsigned long) + 45
6   libchrome_dll.dylib                 0x0000000109650dec importer::ReencodeFavicon(unsigned char const*, unsigned long, std::__Cr::vector<unsigned char, std::__Cr::allocator<unsigned char> >*) + 108
7   libchrome_dll.dylib                 0x000000010738b4ed ChromeImporter::LoadFaviconData(sql::Database*, std::__Cr::map<long long, std::__Cr::set<GURL, std::__Cr::less<GURL>, std::__Cr::allocator<GURL> >, std::__Cr::less<long long>, std::__Cr::allocator<std::__Cr::pair<long long const, std::__Cr::set<GURL, std::__Cr::less<GURL>, std::__Cr::allocator<GURL> > > > > const&, std::__Cr::vector<favicon_base::FaviconUsageData, std::__Cr::allocator<favicon_base::FaviconUsageData> >*) + 557
8   libchrome_dll.dylib                 0x0000000107389d36 ChromeImporter::ImportBookmarks() + 1926
9   libchrome_dll.dylib                 0x0000000107388be4 ChromeImporter::StartImport(importer::SourceProfile const&, unsigned short, ImporterBridge*) + 324
```